### ISSUE TITLE
Enable concurrent runs of grib_to_netcdf worker

### ISSUE THREAD

**Issue by douglatornell** [SalishSeaCast/SalishSeaNowcast#37]
**[Original report](https://bitbucket.org/salishsea/salishseanowcast/issue/37) by Doug Latornell (Bitbucket: [douglatornell](https://bitbucket.org/douglatornell), GitHub: [douglatornell](https://github.com/douglatornell)).**

----------------------------------------

See patch from Michael re:

"The nowcast worker grib_to_netcdf worked and I was able to reprocess the gribs for the west coast project. It takes about a minute per day, and we have ~900 days, and that motivated me to run it in parallel. Doing so required a few edits to the worker, mostly concerning the temporary files (to use unique filenames), and some fiddling with the wgrib2 symlink."

https://bitbucket.org/salishsea/analysis-michael/commits/45f004ff53f842be654a6352756f87ff7a462b09

**Comment by douglatornell** [comment]
Stale issue. No longer relevant after `grib_to_netcdf` refactor in PR #156.
